.scala-steward.conf

```diff
@@ -0,0 +1,115 @@
+# pullRequests.frequency allows to control how often or when Scala Steward
+# is allowed to create pull requests.
+#
+# Possible values:
+#   @asap
+#     PRs are created without delay.
+#
+#   <timespan>
+#     PRs are created only again after the given timespan since the last PR
+#     has passed. Example values are "36 hours", "1 day", or "14 days".
+
+#   <CRON expression>
+#     PRs are created roughly according to the given CRON expression.
+#
+#     CRON expressions consist of five fields:
+#     minutes, hour of day, day of month, month, and day of week.
+#
+#     See https://www.alonsodomin.me/cron4s/userguide/index.html#parsing for
+#     more information about the CRON expressions that are supported.
+#
+#     Note that the date parts of the CRON expression are matched exactly
+#     while the time parts are only used to abide to the frequency of
+#     the given expression.
+#
+# Default: @asap
+#
+#pullRequests.frequency = "0 0 ? * 3" # every thursday on midnight
+pullRequests.frequency = "7 days"
+
+# Only these dependencies which match the given patterns are updated.
+#
+# Each pattern must have `groupId`, and may have `artifactId` and `version`.
+# Defaults to empty `[]` which mean Scala Steward will update all dependencies.
+# updates.allow  = [ { groupId = "com.example" } ]
+
+# The dependencies which match the given version pattern are updated.
+# Dependencies that are not listed will be updated.
+#
+# Each pattern must have `groupId`, `version` and optional `artifactId`.
+# Defaults to empty `[]` which mean Scala Steward will update all dependencies.
+# the following example will allow to update foo when version is 1.1.x
+updates.pin  = [
+  { groupId = "org.apache.spark", artifactId="spark-sql", version = "2.3." },
+  { groupId = "org.apache.spark", artifactId="spark-hive", version = "2.3." },
```

Comment:
I don't think this project compiles any code which directly interacts with `spark-hive` Hive, probably best to leave it agnostic and interface directly with the  default spark-sql warehouse API.

Are there plans to bump / cross-compile for 2.4+ or 3.X?

Reply:
I'll check out dropping the hive dep:

```
$ rg hive
build.sbt
44:  "org.apache.spark" %% "spark-hive" % sparkVersion % Provided,

src/main/resources/log4j-dv-spark.properties
19:log4j.logger.org.apache.hadoop.hive.metastore.RetryingHMSHandler=FATAL
20:log4j.logger.org.apache.hadoop.hive.ql.exec.FunctionRegistry=ERROR
```
I'm going to leave it for now just in case.

Cross-compilation is on my list of things to do but low priority: https://github.com/target/data-validator/issues/84